### PR TITLE
#1879 apply order edit

### LINKED ISSUE #1879: Adding order edit to the jvm SDK > add apply endpoint for order edit

### PR DESCRIPTION
**Pull request by alex-ct**
# Description

Closes #1879 

# Checklist

- [ ] Update release Notes
- [ ] Add to the current github milestone 
- [ ] Update commercetools api reference

### CODE REVIEW

File: commercetools-models/src/main/java/io/sphere/sdk/orderedits/commands/OrderEditApplyCommand.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+public final class OrderEditApplyCommand extends CommandImpl<OrderEdit> {
+
+    private final String id;
+
+    private final Long resourceVersion;
+
+    private final Long editVersion;
+
+    private OrderEditApplyCommand(final String id, final Long resourceVersion, final Long editVersion) {
```
Comment (acbeni) [inline commercetools-models/src/main/java/io/sphere/sdk/orderedits/commands/OrderEditApplyCommand.java]: Here just for clarity reason, name the params as `orderEditId`, `orderEditVersion`, and `orderVersion` in this order,
also add a factory method with parameters `Versioned<OrderEdit>` and `orderVersion`

File: commercetools-models/src/test/java/io/sphere/sdk/orderedit/commands/OrderEditApplyCommandIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import io.sphere.sdk.test.IntegrationTest;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+public class OrderEditApplyCommandIntegrationTest extends IntegrationTest {
+
+    @Test
+    public void applyOrderEdit() {
+        OrderEditFixtures.withUpdateableOrderEdit(client(), orderEdit -> {
+
```
Comment (acbeni) [inline commercetools-models/src/test/java/io/sphere/sdk/orderedit/commands/OrderEditApplyCommandIntegrationTest.java]: Add a check here that the country isnt DE already

File: commercetools-models/src/test/java/io/sphere/sdk/orderedit/commands/OrderEditApplyCommandIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import io.sphere.sdk.orders.queries.OrderByIdGet;
+import io.sphere.sdk.test.IntegrationTest;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+public class OrderEditApplyCommandIntegrationTest extends IntegrationTest {
+
+    @Test
+    public void applyOrderEdit() {
+        OrderEditFixtures.withUpdateableOrderEdit(client(), orderEdit -> {
```
Comment (acbeni) [inline commercetools-models/src/test/java/io/sphere/sdk/orderedit/commands/OrderEditApplyCommandIntegrationTest.java]: add a parameter to `withUpdateableOrderEdit ` of type `order` to make the fixture easier to use.

Comment (alex-ct) [reply commercetools-models/src/test/java/io/sphere/sdk/orderedit/commands/OrderEditApplyCommandIntegrationTest.java]: added a new fixture method that accepts Order as param.

**Pull request merged**